dicom_utils/bot.py

```diff
@@ -0,0 +1,196 @@
+#!/usr/bin/env python
```

Comment:
Should we consider making the name of `bot.py` more descriptive? I.e. `bo_table.py` or just `basic_offset_table.py`?